scripts/creation-portal-migration/template/config/question.js

```diff
@@ -0,0 +1,303 @@
+const questionTemplate = {
```

Comment:
@jayesh9747 Please provide function signatures to clarify what each function does. Make sure to include this for all the functions in your codebase.